<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>376</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Date Change</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font-family: Poppins;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Holiday Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>175</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbHoliday">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>New Year's Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maundy Thursday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Good Friday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Araw ng Kagitingan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Labor Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Independence Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>National Heroes Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bonifacio Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Christmas Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rizal Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese New Year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EDSA Revolution</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black Saturday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ninoy Aquino Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All Saints' Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All Souls' Day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feast of the Immaculate Conception</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Christmas Eve</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Last Day of the Year</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>231</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: &quot;#344273&quot;;
	color: white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string> Update</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../MainFrame/Resources/Icons/update.svg</normaloff>../../MainFrame/Resources/Icons/update.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="sideBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sideBar{
	background-color: qlineargradient(spread:pad, x1:0.508, y1:0.0340909, x2:0.503051, y2:1, stop:0 rgba(33, 44, 89, 255), stop:0.497175 rgba(52, 66, 115, 255), stop:0.847458 rgba(72, 87, 142, 255))
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="DateChange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:&quot;#DCE5FE&quot;;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string>Date Change</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/holiday.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/datetype.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/date.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>231</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: &quot;#344273&quot;;
	color: white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../MainFrame/Resources/Icons/update.svg</normaloff>../../MainFrame/Resources/Icons/update.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDateType">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>Special Holiday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RegularHoliday</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
